pyaedt/generic/LoadAEDTFile.py

```diff
@@ -304,17 +307,16 @@ def _load_entire_aedt_file(filename):
         dictionary containing the decoded AEDT file
 
     """
-    _reaf_aedt_file(filename)
+    global _count
```

Comment:
There must be a way to avoid use of globals.

Reply:
@akaszynski I can easily do it but if I'm going to make that sort of change I'll want to unit test it first.  @maxcapodi78 two questions, 1) how to run cpython unit tests 2) is it ok if the tests for this file are cpython only (I don't think it needs to run in both contexts)

Reply:
Unit test documentation is here:
https://aedtdocs.pyansys.com/Resources/Contributing.html#testing

Our coverage should be testing this.  Check coverage locally by appending `--cov pyaedt --cov-report annotate` to the `pytest` command.

Reply:
global refactor can wait until a different PR since this is out of scope of this PR.

Reply:
@akaszynski 
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --cov --cov-report annotate _unittest
  inifile: C:\AnsysDev\NoBackup\code\ThirdPartySource\PyAEDT\pytest.ini
  rootdir: C:\AnsysDev\NoBackup\code\ThirdPartySource\PyAEDT

Reply:
@koubaa all Unit Tests are in 2 folders _unittest and _unittest_ironpython. If you run on PyCharm simply configure and install pytest and run it from there (right click on folder/file). for UnitTest is slightly different because they run as batch command within AEDT. They both runs on CICD